core/tests/protractor_utils/ProfilePage.js

```diff
@@ -31,6 +31,8 @@ var ProfilePage = function() {
     by.css('.protractor-test-profile-no-interest'));
   var allExplorationCardElements = element.all(
     by.css('.protractor-test-exploration-dashboard-card'));
+  var createdExplorationStat = element.all(
+    by.css('.protractor-test-profile-Created-stat'));
```

Comment:
Typically class names are lower-cased. Can we possibly follow that here?

Reply:
Would have to change the ctrl to have a different name for each of the repeated elements, not sure if that's desired or not

Reply:
I see, you are dynamically creating the class names... Can you possibly add a filter in the js expression generating the class to make it lower case? (I am not very proficient with angular and might be wrong here... but if that is possible, without altering ctrl, that would be desirable).

Reply:
Ah good call, lowercased in the template binding. Done